src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -447,6 +447,99 @@ def get_full_paths(file_list: list) -> list:
     return full_path_files
 
 
+def update_license_file(arg_dict):
+    """
+    Update the LICENSE file to match MIT.txt, adjusting the year span to each repository.
+
+    Parameters
+    ----------
+    arg_dict: dict
+        Dictionary containing the committed files, custom copyright, template, license,
+        changed_headers, start & end year, and git_repo
+    """
+    # Get location of LICENSE file in the repository the hook runs on
+    git_root = arg_dict["git_repo"].git.rev_parse("--show-toplevel")
+    repo_license_loc = os.path.join(git_root, "LICENSE").replace(os.sep, "/")
```

Comment:
```suggestion
    repo_license_loc = os.path.join(git_root, "LICENSE")
```
Why is the replace needed? I don't think it's necessary but I might be wrong

Reply:
I did the replace because git_root always does forward slashes so it would say "successfully changed my/path\LICENSE" if it were windows which looks a little funky

Reply:
Ah gotcha